stats/lib/platform-stats-fetchers.js

```diff
@@ -43,6 +43,20 @@ export const fetchDailyStationAcceptedMeasurementCount = async (pgPool, filter)
   return rows
 }
 
+/**
+ * @param {Queryable} pgPool
+ * @param {import('./typings.js').DateRangeFilter} filter
+ */
+export const fetchTopMeasurementStations = async (pgPool, filter) => {
+  // Ignore the filter for this query
+  // Get the top measurement stations from the Materialized View
+  return (await pgPool.query('SELECT * FROM top_measurement_stations_mv')).rows
+}
+
+export const updateTopMeasurementStations = async (pgPool) => {
+  await pgPool.query('REFRESH MATERIALIZED VIEW top_measurement_stations_mv')
+}
+
```

Comment:
In order to have this reused in both tests and spark-stats.js, I made it a function even though it's one simple line.

I am not sure this is the best place for it, but no where else seemed great. Let me know if you have any thoughts